packages/chain-adapters/src/utxo/UTXOChainAdapter.ts

```diff
@@ -153,9 +158,9 @@ export class ChainAdapter implements IChainAdapter<ChainTypes.Bitcoin> {
         totalPages: data.totalPages,
         transactions: data.transactions.map((tx) => ({
           ...tx,
-          chain: ChainTypes.Bitcoin,
+          chain: this.coinType,
           network: NetworkTypes.MAINNET,
-          symbol: 'BTC',
+          symbol: this.coinName,
```

Comment:
hmm... symbol != coinName. I wonder if this matters. If it does we'll need to look up the symbol from `chainType` or something, and I can't help but wonder what happens if there's more than one coin per chain here?